test/framework/integration_framework/iroha_instance.hpp

```diff
@@ -27,21 +27,35 @@ namespace integration_framework {
 
   using namespace std::chrono_literals;
 
+  /**
+   * Class provides interfaces for running Iroha instance.
+   */
   class IrohaInstance {
    public:
+    /**
+     * Save the block as genesis block.
+     * @param block - block to be saved as genesis block
+     */
     void makeGenesis(const iroha::model::Block &block) {
```

Comment:
The method name is confusing, better to rename it with `insertGenesis` or something similar.